src/ffi/index.ts

```diff
@@ -10,20 +10,40 @@ export const PACT_FFI_VERSION = '0.3.19';
 let ffi: typeof ffiLib;
 let ffiLogLevel: LogLevel;
 
-const initialiseFfi = (logLevel: LogLevel): typeof ffi => {
+const initialiseFfi = (logLevel: LogLevel, logFile?: string): typeof ffi => {
```

Comment:
I think this would be better if it were more explicit - either you have a log file, or you don't.  The current form means that it's easy to misread the intermediate functions that don't touch log level.

The guideline I would recommend is that optional parameters exist only at a boundary - and their presence (or not) is handled in the function where they are optional.

Reply:
This is now moved and updated to 

```
export const getFfiLib = (
  logLevel: LogLevel = DEFAULT_LOG_LEVEL,
  logFile: string | undefined = undefined
): typeof ffi => {
```